code/game/objects/items/storage/pouch.dm

```diff
@@ -93,23 +93,25 @@
 	name = "first-aid pouch"
 	desc = "It can contain autoinjectors, ointments, and bandages."
 	icon_state = "firstaid"
-	storage_slots = 5
+	storage_slots = 6
 	can_hold = list(
 					"/obj/item/stack/medical/ointment",
 					"/obj/item/reagent_container/hypospray/autoinjector/tramadol",
 					"/obj/item/reagent_container/hypospray/autoinjector/tricordrazine",
+					"/obj/item/reagent_container/hypospray/autoinjector/hypervene",
 					"/obj/item/stack/medical/bruise_pack",
 					"/obj/item/stack/medical/splint"
 					)
 
 /obj/item/storage/pouch/firstaid/full
-	desc = "Contains a painkiller autoinjector, first-aid autoinjector, some ointment, and some bandages."
+	desc = "Contains a painkiller autoinjector, first-aid autoinjector, hypervene autoinjector, splints, some ointment, and some bandages."
```

Comment:
This is out of scope.
Again, find a way to introduce your caste without its balance requiring the addition of hypervene to the FAP.

Reply:
Um first of all, it doesn't necessarily require this, it's 

A: an experiment.

and

B: it makes sense. Marines should have countermeasures to known deadly toxins on their person due to the risk involved and their time critical nature.

Reply:
This has been 'fixed'. The growth bonus of growth toxin has been reduced and the autoinjectors have been removed.